src/GraphQL.SystemTextJson/GraphQL.SystemTextJson.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netcoreapp3.1;netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>netstandard2.1;netstandard2.0</TargetFrameworks>
```

Comment:
See https://docs.microsoft.com/en-us/dotnet/standard/net-standard#when-to-target-net50-vs-netstandard

Reply:
Both netcoreapp3.1 and net5 implement netstandard2.1 which contains APIs for spans used here under `#if` preprocessor directives.